flink-core/src/test/java/org/apache/flink/api/java/typeutils/runtime/PojoSerializerTest.java

```diff
@@ -48,10 +47,7 @@
 import java.util.Objects;
 import java.util.Random;
 
-import static org.hamcrest.CoreMatchers.instanceOf;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** A test for the {@link PojoSerializer}. */
 public class PojoSerializerTest extends SerializerTestBase<PojoSerializerTest.TestUserClass> {
```

Comment:
```suggestion
class PojoSerializerTest extends SerializerTestBase<PojoSerializerTest.TestUserClass> {
```
?
\+ there are 3-4 more tests within this class which are public and could be made as package-private (GitHub does not allow to comment on them since they were not touched)